### DISCUSSION TITLE
iOS Memory Issue

### DISCUSSION THREAD

**Discussion by HassanTaleb90** [krzyzanowskim/STTextView#60] [Questions]
When we select all text and then delete it, the process of deleting causes a huge increase in memory usage. After that, if we type a text, the memory increases significantly with each keystroke.

<img width="634" alt="Screenshot 2024-08-11 at 6 46 38 PM" src="https://github.com/user-attachments/assets/e57b71ce-db2c-4948-bc0e-ef9f4a5e577c">

**Comment by krzyzanowskim** [comment]
do you still observe that? I'm not sure I can reproduce it easily on my end. Maybe I'm doing something differently
